packs/feat-effects/effect-draw-from-the-land.json

```
@@ -0,0 +1,41 @@
+{
+    "_id": "zgRWm8KRquDzd1iC",
+    "img": "icons/magic/nature/tree-spirit-green.webp",
+    "name": "Effect: Draw From the Land",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Draw From the Land]</p>\n<p>You gain temporary Hit Points equal to your level. They last for 1 round.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Draw From the Land]</p>\n<p>You gain temporary Hit Points equal to your level.</p>"
```
Duration doesn't need to be in the actual text since its structured